packages/interbit/src/args/getConfig.js

```diff
@@ -13,6 +14,11 @@ const getConfigLocation = () => {
   return configLocation
 }
 
+/**
+ * Gets the interbit configuration file from disk based on the --config
+ * option passed through process args.
+ * @returns {Object|undefined} The configuration file, if found, as a JSON object
```

Comment:
Globally, all descriptions should be written as full sentences, including appropriate punctuation.